docs/robot-playwright.md

```diff
@@ -158,6 +145,108 @@ the user profile. Playwright locators are often much easier to write
 than Selenium locators, which translates to tests and keywords that
 don't have to be tweaked when the page markup changes.
 
+## Writing keywords in Javascript
+
+At its core, playwright is built on node.js, which makes it possible
+to write keywords in Javascript. This is enabled by providing the path
+to a javascript library when the Browser library is
+imported. CumulusCI comes with a small bootstrap file which will
+import a node module which itself can import any number of other
+modules.
+
+```{tip}
+For more information about writing keywords in Javascript see [Extending Browser library with a JavaScript module](
+https://marketsquare.github.io/robotframework-browser/Browser.html#Extending%20Browser%20library%20with%20a%20JavaScript%20module)
+in the [Browser library documentation](https://marketsquare.github.io/robotframework-browser/Browser.html).
+```
+
+### Creating a keyword file in javascript
+
+When you import SalesforcePlaywright.robot into a test, it will import
+the Browser library and pass it the name of a small bootstrap
+javascript file. This bootstrap file will import the file
+`robot/<ProjectName>/javascript/index.js`, exposing the exported
+functions as keywords.
+
+### Example
+
+Who doesn't love a good "Hello, world" example? Start by creating the
+`javascript` folder in your robot folder. For example, if your
+project is named Food-Bank, you should create a folder named
+`robot/Food-Bank/javascript`.
+
+In that folder, create a file named `keywords.js` with the following
+content:
+
+```javascript
+// keywords.js
+async function hello_javascript() {
```

Comment:
Should we provide any guidance around whether to use or not use `async`?

Reply:
I checked with the robot developers and they said it should be OK to not use `async`. The test I wrote works without `async` so I'll remove it, and add a paragraph in the docs about it.